### PR TITLE
Changes to finding wiimotes (again)

### LINKED ISSUE #960: Bug with reconnecting wiimotes as of 99c5e43b586b69006263020ea20dcf69973fb8d3
Hello. I am loving the new wiimote support in Linux. I'm running on 99c5e43b586b69006263020ea20dcf69973fb8d3 for multi-wiimote support. This works nearly perfectly, but there is a bug when remotes reconnect. When wiimotes reconnect, they are detected as new controllers and the old controllers remain.

For an example, say you are playing a 2 player game with remote 1 mapped to real wiimote 1 and remote 2 mapped to real wiimote 2. If wiimote 2 were to disconnect, it would still appear in CEMU. Once the controller reconnects, it would be added as wiimote 3, even though there are still only 2 wiimotes connected. 

Steps to reproduce:
1. Download 99c5e43b586b69006263020ea20dcf69973fb8d3 from here https://github.com/cemu-project/Cemu/actions/runs/5956480086
2. Run CEMU
3. Connect 2 wiimotes
4. Map remote 1 in CEMU to real wiimote 1 and remote 2 in CEMU to real wiimote 2 (should happen automatically)
5. Disconnect wiimote 2 (take a battery out, turn it off, disconnect in bluetooth settings, whatever)
6. Reconnect the wiimote

Expected behavior:
Wiimote 2 is removed when the controller turns off, and is readded when it turns back on

Actual behavior:
Wiimote 2 remains listed when the controller turns off. The wiimote is readded as wiimote 3 when it turns back on. 

This results in controller 2 being mapped to a ghost wiimote, and a "third" wiimote being added. If you were in a multiplayer game, you will have to pause the game and remap the controllers properly before resuming your game. This happened to me a couple times during a four player game, and the result was much confusion trying to remember which wiimotes were ghost wiimotes and which ones were real. There are only 4 wiimotes connected in this image, 2 of which were disconnected mid game
image

### PR DESCRIPTION
**Pull request by capitalistspz**
- Clear `m_wiimotes` before finding new wiimotes
- Use only paths for comparison (I'm pretty sure that it's impossible for multiple wiimotes to open at the same path)
- Also remove a duplicate line from my last PR

Should fix #960

### CODE REVIEW

**Comment by GNUGradyn** [comment]
it looks like the linux build failed https://github.com/cemu-project/Cemu/actions/runs/6111890994/job/16588126558?pr=961

**Comment by GNUGradyn** [comment]
This mostly works. I connected 2 wiimotes to my steam deck and launched the app image generated by this PR. It correctly mapped one of them as player 1 and the other as player 2. The indicators on the remotes matched the mapping in the emulator. However when I disconnected and reconnected player 2, the indicator now shows it's player 1. However, the remote is still correctly mapped to player 2. 

tl;dr everything works as long as you ignore the player indicators when a controller is reconnected
